tools/cli/src/mentat_cli/command_parser.rs

```diff
@@ -90,34 +93,33 @@ pub fn command(s: &str) -> Result<Command, cli::Error> {
 
     let close_parser = string(CLOSE_COMMAND)
                     .with(arguments())
+                    .skip(spaces())
+                    .skip(eof())
                     .map(|args| {
                         if args.len() > 0 {
                             return Err(cli::ErrorKind::CommandParse(format!("Unrecognized argument {:?}", args[0])).into());
```

Comment:
There's a handy `bail!` macro specifically to make this type of early return more pleasant.